### ISSUE TITLE
separate lib from executable

### ISSUE THREAD

**Issue by matthewleon** [purescript/psc-package#53]
Would be nice to have the semantics of `package.json` contained within a separate lib to ease the development of complementary tools.

**Comment by paf31** [comment]
Agreed, thanks for making an issue. Would you be interested in working on this?

**Comment by matthewleon** [comment]
> Would you be interested in working on this?

Sure. Will try to get around to it within a week.
